mods/ts/rules/shared-support.yaml

```diff
@@ -29,7 +29,7 @@ NAPULS:
 		Cost: 1000
 	Tooltip:
 		Name: EMP Cannon
-		Description: Disables vehicles. \nRequires power to operate.\n  Strong vs all ground units\n  Cannot target Aircraft
+		Description: Disables vehicles.\nRequires power to operate.\n  Strong vs all ground units\n  Cannot target Aircraft
```

Comment:
Yes, but the EMP needs some tweaking in general (firing offsets, weapon range etc.), so I'd leave that for its own PR.